server/src/test/java/org/apache/iotdb/db/integration/IoTDBEngineTimeGeneratorIT.java

```diff
@@ -100,7 +101,7 @@ private static void insertData() throws ClassNotFoundException, SQLException {
         Statement statement = connection.createStatement()) {
 
       // create storage group and measurement
-      for (String sql : Constant.create_sql) {
+      for (String sql : org.apache.iotdb.db.constant.TestConstant.create_sql) {
```

Comment:
```suggestion
      for (String sql : TestConstant.create_sql) {
```
And all the same appearance in this test class : )

Reply:
fixed